### DISCUSSION TITLE
Reuse sortable/reorder on custom page?

### DISCUSSION THREAD

**Discussion by Dontorpedo** [filamentphp/filament#4587] [General]
How can i reuse the already installed sortable package on custom page?

i tried it on a custom page with same code like here:

https://livewire-sortablejs.amirrami.com/examples/shared-list

but the elements are not draggable altough it seems that the scripts are loaded.. any hint on that?

**Comment by danharrin** [comment]
`x-sortable` - make the element children sortable
`x-on:end` - event listener for when the sorting is finished

`x-sortable-item` - mark the children
`x-sortable-handle` - mark the draggable handle

**Reply by Dontorpedo** [reply]
hmm, i can't use wire:sortable-group, only wire: sortable works, tried just the example from the site.. but this functionality is crucial to my app.. any idea how i can make it work?

**Reply by danharrin** [reply]
`x-`, not `wire:`

**Reply by danharrin** [reply]
and our implementation doesnt have `-group`
